randovania/data/json_data/prime1.txt

```diff
@@ -54,91 +55,119 @@ Asset id: 2472970646
 Crater Tunnel A
 Asset id: 1238049635
 > Dock to Phazon Core; Heals? False
-  * Normal Door to Phazon Core/Dock to Crater Tunnel A
+  * Plasma Door to Phazon Core/Dock to Crater Tunnel A
 
 > Dock to Crater Entry Point; Heals? False
-  * Normal Door to Crater Entry Point/Dock to Crater Tunnel A
+  * Plasma Door to Crater Entry Point/Dock to Crater Tunnel A
+  > Dock to Phazon Core
+      Space Jump Boots
 
 ----------------
 Phazon Core
 Asset id: 3180620483
 > Dock to Crater Missile Station; Heals? False
-  * Normal Door to Crater Missile Station/Dock to Phazon Core
+  * Plasma Door to Crater Missile Station/Dock to Phazon Core
+  > Dock to Crater Tunnel B
+      Space Jump Boots
```

Comment:
add a connection to Crater Tunnel B with SJ as the requirement

Reply:
is it required to go back to crater tunnel A? since we go to Crater Tunnel B to end the game

Reply:
It doesnt hurt to have, but probably not really required

Reply:
oops forgot about it sorry